src/ogr/geometry.jl

```diff
@@ -43,7 +43,7 @@ Create a geometry object of the appropriate type from its well known text
     geometry to be created. The pointer is updated to point just beyond that
     last character consumed.
 """
-function fromWKT(data::Vector{String})
+function fromWKT(data::Vector{String}; kwargs...)
     geom = Ref{GDALGeometry}()
     result = @gdal(OGR_G_CreateFromWkt::GDAL.OGRErr,
         data::StringList,
```

Comment:
My main question is how best to set `OSRAxisMappingStrategy` for this spatial ref